internal/app/query/query_service.go

```diff
@@ -107,3 +150,108 @@ func getPageSize(cfg config.View) int {
 
 	return pSize
 }
+
+////////////////////////////////////////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////
+
+type ticketCache struct {
```

Comment:
+1 for Saurabh's point. Maybe change the signature to `func (s *queryService) update()` instead?